packages/vsce/README.md

```diff
@@ -1,3 +1,46 @@
-# zowe-native-proto-vsce README
+# VS Code Extension for Zowe Native Protocol
 
-TODO
+This VS Code extension interacts with the Zowe Native Protocol, a native integration with z/OS systems over SSH, allowing developers to interact with mainframe resources directly from their IDE.
+
+## Features
+
+- **Dataset Operations**: Create, read, write, and delete datasets
+- **USS File Access**: Create, read, write, and delete files and directories in Unix System Services
+- **Job Management**: Submit, view, cancel, hold, and release jobs on z/OS
+
+## Minimum Requirements
+
+- VS Code 1.73.0 (or newer)
+- Zowe Explorer v3.1.0 (or newer)
+
+## Installation
+
+Access the latest version of the VS Code extension from the GitHub Releases page.
+
+Install the extension from the VSIX file in VS Code:
+
+1. Open the Extensions view (Ctrl+Shift+X)
+2. Click the `...` button and select `Install from VSIX...`
+3. Select the VSIX file in the `dist` folder
```

Comment:
Similar to CLI - if they're installing from GH release, not sure if we should assume the `dist` folder?